scripts/data/contracts.ts

```diff
@@ -53,6 +53,13 @@ export class ClosePullRequestContribution extends PullRequestContribution {
     }
 }
 
+export class ReviewPullRequestContribution extends PullRequestContribution {
+    constructor(pullrequest: GitPullRequest) {
+        // TODO: Is there a better date to put here than the Closed Date?
```

Comment:
I don't think there is unless you'd prefer the createdDate

Reply:
I think the closed date is a bit more representative of the review date than the created date. Really depends, but let's stick with this.